docs/model-configuration.md

```diff
@@ -57,6 +57,9 @@ The list features only model wrappers which intoduce new configuration values in
 #### `SegmentationModel` and its subclasses
 1. `labels`: List - list of class labels
 1. `path_to_labels`: str - path to file with labels. Overrides the labels, if they sets via 'labels' parameter
+1. `blur_strength`: int - blurring kernel size. -1 signals not to apply bluring and soft_threshold
```

Comment:
`bluring` -> `blurring` everywhere.
`retern` -> `return`
`signals not to apply` sounds not well. I would rephrase to, e.g. `-1 value means no blurring and ...`

Reply:
Applied